app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -1,6 +1,16 @@
 angular.module('miq.util').factory('chartsMixin', function() {
   'use strict';
 
+  var hourlyTimeTooltip = function(d) {
+    var theMoment = moment(d[0].x);
+    return _.template('<table class="c3-tooltip">' +
```

Comment:
curiously enough, it is visible in the bar chart and donuts, but not [sparkline](https://www.patternfly.org/patterns/sparkline/)